Remove `$govuk-show-breakpoints` feature flag (#6384)
alphagov/govuk-frontend: CHANGELOG.md

This change was introduced in [pull request #6366: Update Dart Sass tests minimum version to 1.79.0](https://github.com/alphagov/govuk-frontend/pull/6366)

#### Remove references to the `$govuk-show-breakpoints` feature flag

The `$govuk-show-breakpoints` feature flag was deprecated and rendered non-operational by the media query changes introduced in [GOV.UK Frontend 5.13.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.13.0).

> This flag was only intended to be used as a local development aid and not used on production services. It has now been removed.

Reviewer: ```suggestion
This flag was only intended to be used as a local development aid and not used on production services, and we've now removed it.
```
